test/container/testingScripts/taskSubmission.sh

```diff
@@ -32,20 +32,17 @@ submitTasks(){
 #run immediate check on tasks submitted for Client_Configuration_Validation testing. 
 #Save results to failed_tests and successful_tests files.
 immediateCheck(){
-  project_dir="/tmp/crabTestConfig"
   for task in ${tasksToCheck};
   do
 	echo ${task}
 	test_to_execute=`echo "${task}" | grep -oP '(?<=_crab_).*(?=)'`
-        task_dir=${project_dir}/${test_to_execute}
+	task_dir=`echo "${task}" | grep -oP '(?<=Project dir:\s).*(?=\sPlease)'`
```

Comment:
can you explain a bit in comments what' done here ? can you start from a task name and find a project directory ? or... I am getting lost

Reply:
you are right. commit is bad and will not work. 
`${task}` represents a task name, e.g. `210616_134037:cmsbot_crab_sendExternalFolder`
given that `test_to_execute` will be `sendExternalFolder`
so `task_dir` has to be `task_dir=${project_dir}/crab_${test_to_execute}`